src/pancam.py

```diff
@@ -0,0 +1,22 @@
+from follower import Follower
+
+follower = Follower()
+
+def seek():
+    #tag_data = follower.detect()
+    turn = follower.turn()
+
+    if follower.detect() == False:
+        print('Detect False')
+        follower._speed = follower.LEADER_MAX_SPEED
```

Comment:
What is this code doing? If we don't see the tag, floor it? We should be stopping in this case. And why are you controlling the speed at all in this script? This script is for camera panning, but I don't see camera panning at all in this script.